doc/usecases/gamification/gain_blossoms.md

```diff
@@ -6,14 +6,39 @@
 - **Level:** User Goal
 - **Actors:** App User
 - **Brief:** The user is awarded with Blossoms for achieving certain milestones.
-- **Status:** Draft
+- **Status:** Assigned
+- **Assignee** Thorben
 
 ## Scenarios
 
 - **Precondition:**
   - The user has opened the app and uses it.
 - **Main success scenario:**
-  - The user can see a list of incompleted milestones and their respective requirements, like e.g. planting full groups of companions or collaborating with other users on their map, in the users profile.
+  - The user can see a list of incompleted milestones and their respective requirements. An user can reach a certain milestone by e.g.:
+    - starting their first map.
+    - planting their first plant.
+    - planting their first full group of companions.
+    - planting their first recommended plant.
+    - using an element from a new layer for the first time.
+    - submitting updated plant data.
+    - harvesting a plant for the first time.
+    - becoming a member.
+    - reaching a specific [diversity score](diversity_score.md) goal.
+    - gathering all ingredients for an ingredient list for the first time.
+    - uploading their first photo in the [photo layer](../layers/photo_layer.md).
+    - buying their first batch of seeds.
+    - creating their first event.
+    - honoring a map from another user for the first time.
```

Comment:
Not only "first" but also:
- how often repeated (e.g. once a month for several months)
- several consecutive days
- on special days (e.g. on 12th March https://nationaltoday.com/national-plant-a-flower-day/)
- a specific number within a period of time
- other rules you can think of :wink:

Reply:
I think we should write down the exact milestones in a different document as to not cluster the basic description of the use case.

Reply:
Both is fine: As these are all events for the user, writing them directly in a use case also seems okay to me.